setup.py

```diff
@@ -25,12 +25,14 @@
 version = "1.11.5"
 release_status = "Development Status :: 5 - Production/Stable"
 dependencies = [
+    # Esnure that the lower bounds of these dependencies match what we have in the
```

Comment:
We should automate this then, rather than relying on humans to keep it in sync.

Reply:
Filed https://github.com/googleapis/gapic-generator-python/issues/1777